web/lib/api/graphql/helpers/mappers.ts

```diff
@@ -0,0 +1,95 @@
+import { LlmSchema } from "../../models/requestResponseModel";
+import { HeliconeRequest } from "../../request/request";
+
+export function mapGeminiPro(
```

Comment:
this is hillarious. But also probably doesnt belong in this PR. 

It's fine. just keep it in. just saying it 🤷

Reply:
It is copied from jawn. It just doesn't exist in web atm so wanted to make it the same